examples/mplot3d/trisurf3d_demo.py

```diff
@@ -27,6 +27,6 @@
 fig = plt.figure()
 ax = fig.gca(projection='3d')
 
-ax.plot_trisurf(x, y, z, cmap=cm.jet, linewidth=0.2)
+ax.plot_trisurf(x, y, z, cmap=cm.jet, linewidth=0.2, antialiased=True)
```

Comment:
Ah, right. Because we are taking the edges off of the polygons, the gaps would be apparent. How about we make `antialiased=True` be the default behavior of these functions so that users with existing codebases don't end up with ugly plots.